docs/5g-on-helium/5g-poc.mdx

```diff
@@ -0,0 +1,191 @@
+---
+id: 5g-poc
+sidebar_label: 5G Proof of Coverage
+slug: /5g-on-helium/5g-poc
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+# 5G Proof of Coverage
+
+The Helium blockchain uses a novel work algorithm called “Proof of Coverage” (PoC) to verify that
+Hotspots are located where they claim. Put another way, PoC tries to verify, on an ongoing basis,
+that Hotspots are honestly representing their location and the wireless network coverage they are
+creating from that location.
+
+LoRaWAN Hotspots utilize a PoC method that benefits from Hotspots being used as senders ("Beacon")
+and receivers ("Witness") of PoC activity, you can read more here:
+[Proof of Coverage](./blockchain/proof-of-coverage)
+
+While all current 5G Hotspot models _also_ participate in the LoRaWAN network and PoC, the CBRS
+Small Cell and 5G aspects of a 5G Hotspot require a new approach to validating PoC.
+
+:::info One Way Traffic
+
+CBRS Small Cell Radios can only operate as transmitters, they are not able to receive incoming data
+and thus can not "hear" one another directly.
+
+:::
+
+## 5G PoC Roadmap
+
+- **Stage 1: ** Genesis Rewards
+  - Genesis Phase 1:
+    [Time Period Triggered Rewards](/5g-on-helium/5g-poc#genesis-phase-1-time-period-triggered-rewards)
+    ** (current)**
+  - Genesis Phase 2:
+    [Consensus Election Triggered Rewards](/5g-on-helium/5g-poc#genesis-phase-2-consensus-election-triggered-rewards)
+  - Genesis Phase 3:
+    [Speed Test and Quality of Service](/5g-on-helium/5g-poc#genesis-phase-3-speed-test-and-quality-of-service)
+  - Genesis Phase 4:
+    [Reward Frequency Increase](/5g-on-helium/5g-poc#genesis-phase-4-reward-frequency-increase)
+  - Genesis Phase 5:
+    [Reward Period Decoupling From Reward Submission Time](/5g-on-helium/5g-poc#genesis-phase-5-reward-period-decoupling)
+- **Stage 2: ** 5G Mappers Proof of Coverage
+- **Stage 3: ** Additional Methods of Proof of Coverage
+
+---
+
+## Current 5G PoC Reward Elegibility
+
+MOBILE Rewards are earned by a Wallet with a 5G Hotspot and at least one connected, SAS registered,
+CBRS Small Cell Radio in one "lump sum" every roughly 24-hour interval. This system allows a high
+tolerance for downtime in infrastructure that is beyond the 5G Hotspot Owner's control.
+
+To qualify for the Genesis Rewards in a Reward Period, a CBRS Small Cell Radio must:
+
+- Be connected to a 5G Hotspot.
+- Be properly installed.
+  - Be registered with the SAS (Outdoor Radios Only).
+- Broadcast at least 1 "heartbeat" per Reward Period.
+
+:::note What is a "Heartbeat"?
+
+A "heartbeat" is an authorization that allows the Small Cell Radio to transmit 5G coverage.
```

Comment:
I think it is confusing to call "heartbeat" an authorization. I propose something like "Heartbeat" is data sent by Hotspot indicating that the connected Small Cell Radio(s) is authorized to transmit 5G coverage.

Reply:
much more clear, fixed